app/models/spotlight/blacklight_configuration.rb

```diff
@@ -315,13 +315,22 @@ def add_exhibit_specific_fields(config)
     def add_exhibit_tags_fields(config)
       # rubocop:disable Style/GuardClause
       unless config.show_fields.include? :exhibit_tags
-        config.add_show_field :exhibit_tags, field: config.document_model.solr_field_for_tagger(exhibit),
-                                             link_to_facet: true,
-                                             separator_options: { words_connector: nil, two_words_connector: nil, last_word_connector: nil }
+        config.add_show_field(
+          :exhibit_tags,
+          field: config.document_model.solr_field_for_tagger(exhibit),
+          link_to_facet: true,
+          separator_options: { words_connector: nil, two_words_connector: nil, last_word_connector: nil },
+          label: I18n.t('spotlight.search.fields.facet.exhibit_tags')
```

Comment:
@jeremyf I think the show field should use the non-facet locale -- 

```suggestion
          label: I18n.t('spotlight.search.fields.exhibit_tags')
```